fabric-content-registries-v0/src/main/resources/fabric-content-registries-v0.mixins.json

```diff
@@ -3,8 +3,8 @@
   "package": "net.fabricmc.fabric.mixin.content.registry",
   "compatibilityLevel": "JAVA_8",
   "mixins": [
-    "MixinAbstractFurnaceBlockEntity",
-    "MixinFireBlock"
+    "LegacyMixinAbstractFurnaceBlockEntity",
+    "LegacyMixinFireBlock"
```

Comment:
Why rename the old classes? they are in a diffrent package.

Reply:
mixin and impl aren't in different packages. Presumably it's like that because they can change at any time and aren't fixed like API is

Reply:
Is there a specific reason you chose to isolate the deprecated and new versions, rather than just making the old version depend on V1, and then just redirect the v0 calls into V1 registries

Reply:
Yeah, because the old API doesn't translate to the new one perfectly. The old .clear method is gone (explanation for that in PR description).